### PR TITLE
`treehouses magazine` (fixes #1274)

### LINKED ISSUE #1274: `treehouses magazine`?
`treehouses magazine magpi [all|latest|number]` for downloading https://magpi.raspberrypi.org/

e.g. https://magpi.raspberrypi.org/issues/93/pdf
     https://magpi.raspberrypi.org/issues

### PR DESCRIPTION
**Pull request by jayala-29**
fixes #1274

`treehouses magazine` module

### CODE REVIEW

**Comment by jayala-29** [comment]
currently the following commands are functional:
- `treehouses magazine magpi`
- `treehouses magazine magpi latest`
- `treehouses magazine magpi [number]`

**Comment by jayala-29** [comment]
Screen Shot 2020-05-22 at 4 28 26 PM

**Comment by jayala-29** [comment]
Screen Shot 2020-05-22 at 5 18 10 PM

**Comment by jayala-29** [comment]
Screen Shot 2020-05-22 at 5 50 16 PM

File: modules/help.sh [hunk 1]
```diff
@@ -108,6 +108,7 @@ convert <input file> <output file>        converts video and audio files
 gpio                                      displays raspberry pi model and corresponding GPIO ports
 changelog                                 displays the most recent changes to treehouses
 shutdown [now|in|force]                   shutdown the system
+magazine magpi [all|latest|number]        downloads specific magpi issue as a pdf
```
Comment (law-lin) [inline modules/help.sh:111]: I remember @dogi suggested me to leave `shutdown` to be the very last, not a serious issue though.

File: README.md [hunk 2]
```diff
@@ -132,6 +132,7 @@ convert <input file> <output file>        converts audio and video files
 gpio                                      displays raspberry pi model and corresponding GPIO ports
 changelog [view|compare]                  displays the most recent changes to treehouses
 shutdown [now|in|force]                   shutdown the system
+magazine magpi [all|latest|number]        downloads specific magpi issue as a pdf
```
Comment (law-lin) [inline README.md:135]: Same here, `shutdown` should be last.

**Comment by LordJashin32** [comment]
7 variables. What a crazy guy. Do we have a way to view PDF files on Pi?

**Pull request merged**